python/tvm/meta_schedule/logging.py

```diff
@@ -62,15 +62,15 @@ def get_logging_func(logger: Logger) -> Optional[Callable[[int, str], None]]:
         # logging.FATAL not included
     }
 
-    def logging_func(level: int, msg: str):
-        if level < 0:
+    def logging_func(level: int, file: str, lineo: int, msg: str):
```

Comment:
Do you think `file_name` is a better name?
```suggestion
    def logging_func(level: int, file_name: str, lineo: int, msg: str):
```

Reply:
i think "filename" is probably a more frequent word (?)

Reply:
I think you're right! Thanks for pointing it out!